src/components/pages/TeachersPage/index.tsx

```diff
@@ -141,6 +157,7 @@ export default function TeachersPage({ classroomName }: ClassroomProps) {
             studentChats={studentChats}
             setDisplayedChat={setDisplayedChat}
             displayedChat={displayedChat}
+            unPair={unPair}
```

Comment:
Throughput this page, unpair was unpair.  So P wasn't capitalized. Let's keep it that way for consistency.
```suggestion
            unpair={unpair}
```